src/app/file-browser/components/edit-tags/edit-tags.component.ts

```diff
@@ -157,6 +160,12 @@ export class EditTagsComponent implements OnInit, DoCheck, OnDestroy, HasSubscri
 
     this.itemTagsById.clear();
 
+    this.itemTags = this.item.TagVOs.map((tag) => {
+      return this.allTags.filter(
+        (t) => t.tagId === tag.tagId
+      )[0];
+    });
```

Comment:
Could this be rewritten more simply with [`Array.find`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find)?

```suggestion
    this.itemTags = this.item.TagVOs.map(
      (tag) => this.allTags.find((t) => t.tagId === tag.tagId)
    );
```

Alternatively, it might make more sense to store these in `Map`s rather than `Array`s, if this is a frequent operation.